build/audit.sh

```diff
@@ -31,7 +31,9 @@ SELF=audit
 
 . ./common.sh
 
-if [ -z "${PORTSLIST}" ]; then
+if [ "${PORTSLIST}" ]; then
+	PORTSLIST=$(multi_line_list "${PORTSLIST}")
+else
 	PORTSLIST=$(list_packages ${CONFIGDIR}/ports.conf)
```

Comment:
Sorry to derail this... how about making this simpler? we would save the full if construct and could push conversion down to list_packages() where it used to be?

```suggestion
	PORTSLIST=$(list_packages "${PORTSLIST}" ${CONFIGDIR}/ports.conf)
```